tests/test_api_roles.py

```diff
@@ -618,6 +620,338 @@ def test_08_user_can_set_pin(self):
             self.assertTrue(response.get("result").get("value"),
                             response.get("result"))
 
+    def test_09_authz_user_detail(self):
+        # Test behavior of ADDUSERINRESPONSE and ADDRESOLVERINRESPONSE policy action for /auth endpoint
+        # Normally, no userinfo and realm/resolver are found in the response
+        with self.app.test_request_context('/auth',
+                                           method='POST',
+                                           data={"username": "selfservice@realm1",
+                                                 "password": "test"}):
+            res = self.app.full_dispatch_request()
+            self.assertTrue(res.status_code == 200, res)
+            content = json.loads(res.data.decode('utf8'))
```

Comment:
I'll change the json stuff and push it here.

Reply:
thx